smartcar/smartcar.py

```diff
@@ -85,7 +85,7 @@ def __init__(self, client_id, client_secret, redirect_uri, scope=None, test_mode
         else:
             self.test_mode = test_mode if test_mode else False
 
-    def get_auth_url(self, force=False, state=None, vehicle_info=None):
+    def get_auth_url(self, force=False, state=None, vehicle_info=None, single_select=False):
```

Comment:
should default to none right?

Reply:
yup, I think I didn't catch this when I rebased! fixed!